test/spec/dataviews/category-dataview-model.spec.js

```diff
@@ -146,6 +146,34 @@ describe('dataviews/category-dataview-model', function () {
     expect(this.model._fetch.calls.count()).toEqual(1);
   });
 
+  describe('filters over data', function () {
```

Comment:
Ok!, but let me do that inside a describe that says the purpose of those functions.